server_environment_delivery/models/delivery_carrier.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import models
+
+
+class DeliveryCarrier(models.Model):
+    _name = "delivery.carrier"
+    _inherit = ["delivery.carrier", "server.env.mixin"]
```

Comment:
I'd recommend to use the brand new mixin `server.env.techname.mixin` because the carrier name could change based on whatever needs.